tests/tensor/test_variable.py

```diff
@@ -451,6 +451,22 @@ def test_set_item_error(self):
         with pytest.raises(TypeError, match=msg):
             x[0] += 5
 
+    def test_transpose(self):
+        X, _ = self.vars
+        x, _ = self.vals
+
+        # Turn (2,2) -> (1,2)
+        X, x = X[1:, :], x[1:, :]
+
+        assert_array_equal(X.transpose(0, 1).eval({X: x}), x.transpose(0, 1))
+        assert_array_equal(X.transpose(1, 0).eval({X: x}), x.transpose(1, 0))
+
+        # Test handing in lists and np.arrays
+        assert_array_equal(X.transpose([1, 0]).eval({X: x}), x.transpose([1, 0]))
```

Comment:
No need to eval these, having confirmed the above works, you can use `assert equal_computations([X.tranpose(array(...))], [X.transposr(list(...))])` that confirms you have the expected graph

Reply:
Not sure how checking them at the same time would look like, but fixed code to do equal_computation now. 
Also added the tuple alongside list and np.array as that was the third option

Reply:
you can have multiple variables in each side of `equal_computations`, such as `equal_computations([var1, var2], [var3, var4])`, which will check if var1 is equal to var3 and var2 to var4